nixos/tests/postgresql.nix

```diff
@@ -1,26 +1,51 @@
-import ./make-test.nix ({ pkgs, ...} : {
+import ./make-test.nix ({ pkgs, ...} : 
+let
+  postgresql-versions = [
```

Comment:
If anyone knows how to better parametrize this test please let me know. would be nice to pull posgresql param out of the test function so tests can be run in parallel.